atomic_scanners/pipeline-scanner/pipeline-scanner

```diff
@@ -3,8 +3,8 @@ scanner_name: pipeline-scanner
 image_name: registry.centos.org/pipeline-images/pipeline-scanner
 default_scan: redhat-release
 scans: [
-  { name: redhat-release,
+  { name: pipeline-scanner,
```

Comment:
https://github.com/CentOS/container-pipeline-service/blob/master/beanstalk_worker/worker_start_scan.py#L346 , if we change the name, worker wont be able to find the scanner.